doc/source/options.rst

```diff
@@ -142,6 +142,11 @@ tag-format = a string
     It needs to contain ``{version}``.
     For backward compatibility, it can contain ``%(version)s`` instead.
 
+tag-signing = yes / no
+    Default: no.
+    When set to true, tags are signed using the signing feature of the
+    respective vcs. Currently tag-signing is only supported for git.
```

Comment:
Perhaps add a short note like this: "Note: when you enable it, you require that everyone that releases your project has git tag signing set up correctly."

On the other hand, perhaps it is obvious. @mauritsvanrees ?

Reply:
I'll add this. Even if it's obvious for most, it might help those for those it it not :-)